.gitignore

```diff
@@ -148,4 +148,7 @@ target
 hs_err_pid*
 replay_pid*
 
-# End of https://www.toptal.com/developers/gitignore/api/kotlin
\ No newline at end of file
+# End of https://www.toptal.com/developers/gitignore/api/kotlin
+
+## Firebase Key
+/src/main/resources/serviceAccountKey.json
```

Comment:
Could help send template for serviceAccountkey.json into the discord?

Reply:
For this one can download from firebase. I will create a document for this one.